feature_engine/selection/feature_selection.py

```diff
@@ -0,0 +1,176 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import get_scorer
```

Comment:
we are not using get_scorer, no? can we remove? also, are we using np? otherwise please also remove from theimports